sdk/formrecognizer/Azure.AI.FormRecognizer/Azure.AI.FormRecognizer-Preview2.sln

```diff
@@ -7,6 +7,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.AI.FormRecognizer", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.AI.FormRecognizer.Tests", "tests\Azure.AI.FormRecognizer.Tests.csproj", "{4F476D56-DDE7-43D3-8CB4-BA1E77F5A300}"
```

Comment:
Just curious, the renaming is so that locally you could have both projects, but once decided what P1 is going to have this rename won't be necessary?

Reply:
Yes, I did this so I could have both Preview 1 and Preview 2 solutions open next to each other side by side and I wouldn't be confused about which is which.  We'll definitely put this back when it's merged back to master.